src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -27,6 +34,42 @@ internal class RPackageManager : IRPackageManager {
             return VisualComponent;
         }
 
+        public async Task<IList<RPackage>> GetInstalledPackagesAsync() {
+            if (_interactiveWorkflow.RSession == null || !_interactiveWorkflow.RSession.IsHostRunning) {
+                return new List<RPackage>();
+            }
+
+            REvaluationResult result;
+            using (IRSessionEvaluation eval = await _interactiveWorkflow.RSession.BeginEvaluationAsync()) {
```

Comment:
If I use RSession.EvaluateAsync to implement GetInstalledPackages and GetAvailablePackages, I'm having trouble in the tests where the test never calls RSession.BeginEvaluationAsync.  The trouble is that it complains that `rtvs` package doesn't exist (it's used by the code I'm evaluating).
The tests that do setup work that still uses BeginEvaluationAsync have no problem when they get to call GetInstalledPackages.

So is this a gotcha of the "you don't care when it happens" ?